dodola/core.py

```diff
@@ -297,6 +297,20 @@ def standardize_gcm(ds, leapday_removal=True):
     )
 
     # Cleanup time.
+
+    # if variable is precip, need to update units to mm/day
+    if "pr" in ds_cleaned.variables:
+        # units should be kg/m2/s in CMIP6 output
+        if ds_cleaned["pr"].units == "kg m-2 s-1":
+            # convert to mm/day
+            mmday_conversion = 24 * 60 * 60
+            ds_cleaned["pr"] = ds_cleaned["pr"] * mmday_conversion
+            # update units attribute
+            ds_cleaned["pr"].attrs["units"] = "mm/day"
+        else:
+            # we want this to fail, as pr units are something we don't expect
+            raise AssertionError("check units: pr units attribute is not kg m-2 s-1")
```

Comment:
This should be a custom exception or a `ValueError`.

As a rule, assertions are used to check for a condition that should never happen. This is checking for an inappropriate arg value, at least.

Reply:
Ahh yes. Thanks - updated this.